src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -1,82 +1,20 @@
-import akka.stream.Materializer
-import akka.stream.scaladsl.Sink
-import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.dsde.workbench.avram.errorReportSource
-import org.broadinstitute.dsde.workbench.avram.model.AvramJsonSupport._
-import org.broadinstitute.dsde.workbench.model.ErrorReportJsonSupport._
-import org.broadinstitute.dsde.workbench.model.{ErrorReport, WorkbenchException, WorkbenchExceptionWithErrorReport}
 
-import scala.concurrent.{ExecutionContext, Future}
-import scala.concurrent.{ExecutionContext, Future}
+case class Pong()
 
-class AvramRoutes()(implicit val system: ActorSystem, val materializer: Materializer, val executionContext: ExecutionContext)
-  extends LazyLogging {
 
-  def avramRoutes: server.Route =
-    pathPrefix("ping") {
-      pathEndOrSingleSlash {
-        get {
-          complete {
-            StatusCodes.OK
-          }
-        }
-      }
-    }
+@Api(name = "avram", version = "v1", scopes = Array("https://www.googleapis.com/auth/userinfo.email"))
+class AvramRoutes {
 
-  def route: server.Route = (logRequestResult & handleExceptions(myExceptionHandler)) {
-    avramRoutes
-  }
-
-  private val myExceptionHandler = {
-
-    ExceptionHandler {
-      //case withErrorReport: WorkbenchExceptionWithErrorReport =>
-      //  complete(withErrorReport.errorReport.statusCode.getOrElse(StatusCodes.InternalServerError), withErrorReport.errorReport)
-      case e: Throwable =>
-        //NOTE: this needs SprayJsonSupport._, ErrorReportJsonSupport._, and errorReportSource all imported to work
-        complete(StatusCodes.InternalServerError, ErrorReport(e))
-    }
-  }
+  private val log = new Logger
 
-  // basis for logRequestResult lifted from http://stackoverflow.com/questions/32475471/how-does-one-log-akka-http-client-requests
-  private def logRequestResult: Directive0 = {
-    def entityAsString(entity: HttpEntity): Future[String] = {
-      entity.dataBytes
-        .map(_.decodeString(entity.contentType.charsetOption.getOrElse(HttpCharsets.`UTF-8`).value))
-        .runWith(Sink.head)
-    }
-
-    def myLoggingFunction(logger: LoggingAdapter)(req: HttpRequest)(res: Any): Unit = {
-      val entry = res match {
-        case Complete(resp) =>
-          val logLevel: LogLevel = resp.status.intValue / 100 match {
-            case 5 => Logging.ErrorLevel
-            case _ => Logging.DebugLevel
-          }
-          entityAsString(resp.entity).map(data => LogEntry(s"${req.method} ${req.uri}: ${resp.status} entity: $data", logLevel))
-        case other =>
-          Future.successful(LogEntry(s"$other", Logging.DebugLevel)) // I don't really know when this case happens
-      }
-      entry.map(_.logTo(logger))
-    }
-
-    DebuggingDirectives.logRequestResult(LoggingMagnet(log => myLoggingFunction(log)))
+  @ApiMethod(name = "ping", httpMethod = "get", path = "ping")
+  def ping: Pong = {
+    log.log("HIII")
```

Comment:
HI!!1

Reply:
Haha, testing the logging - I'll remove it